.github/CODEOWNERS

```diff
@@ -183,21 +182,6 @@ app/models/backend_status.rb @department-of-veterans-affairs/lighthouse-banana-p
-app/models/bgs_dependents/child_school.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/child_stopped_attending_school.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/child_student.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/death.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/divorce.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/marriage_history.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/relationship.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/spouse.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/step_child.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/veteran.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/bgs_dependents/vnp_benefit_claim.rb @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/benefits-dependents-management @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
Are all of these lines being deleted because `app/models/bgs_dependents` will take care of everything?